teamcity: failed tests on master: test/TestLogic, test/TestLogic/distsql, test/TestLogic/distsql/cascade, test/TestLogic/distsql/cascade/DeleteCascade_Multi, acceptance/TestNodeRestart, acceptance/TestNodeRestart/runMode=local, acceptance/TestNodeRestart, acceptance/TestNodeRestart/runMode=local, test/TestLogic, test/TestLogic/distsql, test/TestLogic/distsql/cascade, test/TestLogic/distsql/cascade/DeleteCascade_Multi

The following tests appear to have failed:

[#500417](https://teamcity.cockroachdb.com/viewLog.html?buildId=500417):


```
--- FAIL: test/TestLogic (65.220s)
test_log_scope.go:81: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic224714381
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: test/TestLogic/distsql (0.240s)
None
--- FAIL: test/TestLogic/distsql/cascade (3.330s)
None
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_Multi (0.040s)
logic.go:2069: 
	testdata/logic_test/cascade:890: SELECT id, a_id FROM b;
	expected:
	    2  NULL
	    5  NULL
	    
	but found (query options: "") :
	    5  NULL  
	    2  NULL  
	
logic.go:2069: testdata/logic_test/cascade:896: too many errors encountered, skipping the rest of the input
------- Stdout: -------
--- done: testdata/logic_test/parallel_stmts with config distsql: 99 tests, 0 failures
E180131 15:08:38.941212 284167 sql/logictest/logic.go:2067  
testdata/logic_test/cascade:890: SELECT id, a_id FROM b;
expected:
    2  NULL
    5  NULL
    
but found (query options: "") :
    5  NULL  
    2  NULL  
E180131 15:08:38.941316 284167 sql/logictest/logic.go:2067  testdata/logic_test/cascade:896: too many errors encountered, skipping the rest of the input
--- done: testdata/logic_test/cascade with config distsql: 131 tests, 2 failures
--- done: testdata/logic_test/rename_table with config distsql: 66 tests, 0 failures
--- done: testdata/logic_test/orms with config distsql: 13 tests, 0 failures
--- done: testdata/logic_test/alias_types with config distsql: 10 tests, 0 failures
--- done: testdata/logic_test/pgoidtype with config distsql: 61 tests, 0 failures
--- done: testdata/logic_test/snapshot_timestamp_drift with config distsql: 9 tests, 0 failures
--- done: testdata/logic_test/postgresjoin with config distsql: 101 tests, 0 failures
--- done: testdata/logic_test/name_escapes with config distsql: 6 tests, 0 failures
--- done: testdata/logic_test/rename_database with config distsql: 27 tests, 0 failures
--- done: testdata/logic_test/needed_columns with config distsql: 22 tests, 0 failures
--- done: testdata/logic_test/namespace with config distsql: 7 tests, 0 failures
--- done: testdata/logic_test/order_by with config distsql: 106 tests, 0 failures
--- done: testdata/logic_test/limit with config distsql: 13 tests, 0 failures
--- progress: testdata/logic_test/alter_table: 56 statements/queries
--- done: testdata/logic_test/no_primary_key with config distsql: 17 tests, 0 failures
--- done: testdata/logic_test/multi_statement with config distsql: 14 tests, 0 failures
--- done: testdata/logic_test/rename_column with config distsql: 22 tests, 0 failures
--- done: testdata/logic_test/json_builtins with config distsql: 175 tests, 0 failures
--- done: testdata/logic_test/json with config distsql: 91 tests, 0 failures
--- done: testdata/logic_test/views with config distsql: 114 tests, 0 failures
--- total progress: 18895 statements/queries
--- done: testdata/logic_test/manual_retry with config distsql: 8 tests, 0 failures
--- FAIL: acceptance/TestNodeRestart (9.510s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestNodeRestart517543585
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestNodeRestart/runMode=local (9.500s)
zchaos_test.go:152: pq: rpc error: code = Unavailable desc = all SubConns are in TransientFailure
------- Stdout: -------
CockroachDB node starting at 2018-01-31 15:17:45.395014157 +0000 UTC (took 0.6s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:39933
sql:                 postgresql://root@127.0.0.1:42451?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/cockroach-temp865725002
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1
status:              initialized new cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              1
CockroachDB node starting at 2018-01-31 15:17:45.994116023 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:43845
sql:                 postgresql://root@127.0.0.1:41629?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2/cockroach-temp346044033
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              2
CockroachDB node starting at 2018-01-31 15:17:45.997004516 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:36367
sql:                 postgresql://root@127.0.0.1:44053?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4/cockroach-temp520874065
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              3
CockroachDB node starting at 2018-01-31 15:17:46.002303475 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:38787
sql:                 postgresql://root@127.0.0.1:34837?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp827654121
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
CockroachDB node starting at 2018-01-31 15:17:48.960749888 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34209
sql:                 postgresql://root@127.0.0.1:42385?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp030826057
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
E180131 15:17:50.188999 4003 acceptance/zchaos_test.go:272  round 1: failed to do consistency check against node 3: Consistency checking is unimplmented and should be re-implemented using SQL
CockroachDB node starting at 2018-01-31 15:17:50.907800443 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34639
sql:                 postgresql://root@127.0.0.1:46399?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp773623271
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
E180131 15:17:52.233905 4003 acceptance/zchaos_test.go:272  round 2: failed to do consistency check against node 3: Consistency checking is unimplmented and should be re-implemented using SQL
CockroachDB node starting at 2018-01-31 15:17:52.993990719 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34403
sql:                 postgresql://root@127.0.0.1:33581?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/cockroach-temp091970557
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              1
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestNodeRestart517543585
--- FAIL: acceptance/TestNodeRestart (9.510s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestNodeRestart517543585
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestNodeRestart/runMode=local (9.500s)
zchaos_test.go:152: pq: rpc error: code = Unavailable desc = all SubConns are in TransientFailure
------- Stdout: -------
CockroachDB node starting at 2018-01-31 15:17:45.395014157 +0000 UTC (took 0.6s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:39933
sql:                 postgresql://root@127.0.0.1:42451?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/cockroach-temp865725002
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1
status:              initialized new cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              1
CockroachDB node starting at 2018-01-31 15:17:45.994116023 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:43845
sql:                 postgresql://root@127.0.0.1:41629?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2/cockroach-temp346044033
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/2
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              2
CockroachDB node starting at 2018-01-31 15:17:45.997004516 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:36367
sql:                 postgresql://root@127.0.0.1:44053?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4/cockroach-temp520874065
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/4
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              3
CockroachDB node starting at 2018-01-31 15:17:46.002303475 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:38787
sql:                 postgresql://root@127.0.0.1:34837?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp827654121
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              initialized new node, joined pre-existing cluster
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
CockroachDB node starting at 2018-01-31 15:17:48.960749888 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34209
sql:                 postgresql://root@127.0.0.1:42385?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp030826057
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
E180131 15:17:50.188999 4003 acceptance/zchaos_test.go:272  round 1: failed to do consistency check against node 3: Consistency checking is unimplmented and should be re-implemented using SQL
CockroachDB node starting at 2018-01-31 15:17:50.907800443 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34639
sql:                 postgresql://root@127.0.0.1:46399?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/cockroach-temp773623271
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/3
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              4
E180131 15:17:52.233905 4003 acceptance/zchaos_test.go:272  round 2: failed to do consistency check against node 3: Consistency checking is unimplmented and should be re-implemented using SQL
CockroachDB node starting at 2018-01-31 15:17:52.993990719 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-151-gc55cd0e @ 2018/01/31 15:07:01 (go1.9.3)
admin:               http://127.0.0.1:34403
sql:                 postgresql://root@127.0.0.1:33581?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestNodeRestart/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/cockroach-temp091970557
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster558365068/1
status:              restarted pre-existing node
clusterID:           8a0c6bb6-cbe0-46cd-8dfb-4ca010483f3c
nodeID:              1
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestNodeRestart517543585
--- FAIL: test/TestLogic (65.220s)
test_log_scope.go:81: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic224714381
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: test/TestLogic/distsql (0.240s)
None
--- FAIL: test/TestLogic/distsql/cascade (3.330s)
None
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_Multi (0.040s)
logic.go:2069: 
	testdata/logic_test/cascade:890: SELECT id, a_id FROM b;
	expected:
	    2  NULL
	    5  NULL
	    
	but found (query options: "") :
	    5  NULL  
	    2  NULL  
	
logic.go:2069: testdata/logic_test/cascade:896: too many errors encountered, skipping the rest of the input
------- Stdout: -------
--- done: testdata/logic_test/parallel_stmts with config distsql: 99 tests, 0 failures
E180131 15:08:38.941212 284167 sql/logictest/logic.go:2067  
testdata/logic_test/cascade:890: SELECT id, a_id FROM b;
expected:
    2  NULL
    5  NULL
    
but found (query options: "") :
    5  NULL  
    2  NULL  
E180131 15:08:38.941316 284167 sql/logictest/logic.go:2067  testdata/logic_test/cascade:896: too many errors encountered, skipping the rest of the input
--- done: testdata/logic_test/cascade with config distsql: 131 tests, 2 failures
--- done: testdata/logic_test/rename_table with config distsql: 66 tests, 0 failures
--- done: testdata/logic_test/orms with config distsql: 13 tests, 0 failures
--- done: testdata/logic_test/alias_types with config distsql: 10 tests, 0 failures
--- done: testdata/logic_test/pgoidtype with config distsql: 61 tests, 0 failures
--- done: testdata/logic_test/snapshot_timestamp_drift with config distsql: 9 tests, 0 failures
--- done: testdata/logic_test/postgresjoin with config distsql: 101 tests, 0 failures
--- done: testdata/logic_test/name_escapes with config distsql: 6 tests, 0 failures
--- done: testdata/logic_test/rename_database with config distsql: 27 tests, 0 failures
--- done: testdata/logic_test/needed_columns with config distsql: 22 tests, 0 failures
--- done: testdata/logic_test/namespace with config distsql: 7 tests, 0 failures
--- done: testdata/logic_test/order_by with config distsql: 106 tests, 0 failures
--- done: testdata/logic_test/limit with config distsql: 13 tests, 0 failures
--- progress: testdata/logic_test/alter_table: 56 statements/queries
--- done: testdata/logic_test/no_primary_key with config distsql: 17 tests, 0 failures
--- done: testdata/logic_test/multi_statement with config distsql: 14 tests, 0 failures
--- done: testdata/logic_test/rename_column with config distsql: 22 tests, 0 failures
--- done: testdata/logic_test/json_builtins with config distsql: 175 tests, 0 failures
--- done: testdata/logic_test/json with config distsql: 91 tests, 0 failures
--- done: testdata/logic_test/views with config distsql: 114 tests, 0 failures
--- total progress: 18895 statements/queries
--- done: testdata/logic_test/manual_retry with config distsql: 8 tests, 0 failures

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/cascade
+++ b/pkg/sql/logictest/testdata/logic_test/cascade
@@ -1778,7 +1778,7 @@ INSERT INTO b VALUES (1, 1), (2, NULL), (3, 2), (4, 1), (5, NULL);
 statement ok
 UPDATE a SET id = id + 10;
 
-query II
+query II rowsort
 SELECT id, a_id FROM b;
 ----
 1  11

